[feature] Added environment variables for Redis configuration #463 (#471)
openwisp/docker-openwisp: docs/user/settings.rst

- **Valid Values:** ``True``, ``False``.
- **Default:** ``False``.

``REDIS_CACHE_URL``
~~~~~~~~~~~~~~~~~~~

> - **Explanation:** Redis database URL for Django cache.

Reviewer: ```suggestion
- **Explanation:** Allows freely redefining the Redis database URL for the Django cache.
```

- **Valid Values:** STRING.
- **Default:**
  ``redis://<REDIS_USER>:<REDIS_PASS>@<REDIS_HOST>:<REDIS_PORT>/0``.

``CHANNEL_REDIS_URL``
~~~~~~~~~~~~~~~~~~~~~

> - **Explanation:** Redis database URL for Django channel's layer.

Reviewer: ```suggestion
- **Explanation:** Allows freely redefining the Redis database URL for Django Channels' layer.
```

- **Valid Values:** STRING.
- **Default:**
  ``redis://<REDIS_USER>:<REDIS_PASS>@<REDIS_HOST>:<REDIS_PORT>/1``.

``CELERY_BROKER_URL``
~~~~~~~~~~~~~~~~~~~~~

> - **Explanation:** Redis database URL for celery broker.

Reviewer: ```suggestion
- **Explanation:** Allows freely redefining the Redis database URL for the Celery broker.
```
